### ISSUE TITLE
[WCAG] WCAG Issues

### ISSUE THREAD

**Issue by jolevesq** [Canadian-Geospatial-Platform/geoview#881]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

There is many WCAG issue

- [x] Hidden tab. When we tab trough the viewer, we loose focus and do not know where we are.
- [x] The cross hair and message for map navigation only appears the first time we cycle trough the map
- [x] The app bar selector state stay on when we close the panel with space bar (focus on X) or ESC key
- [x] When more then one  map with footer bar, click space bar on legend icons move the the other map tabs
- [x] When we tab to the legend icon stack, there is tabbing between the stack element and only the first one expand the stack. It should be only one element
- [x] When we expand/collapse the minimap with the keyboard, we loose the focus outline around the button. We should always have the outline focus when using the keyboard. __This applies to other map element like Nav-Bar and Footer Bar__
- [ ] When we expand the legend stack icon, we loose focus and can't close it
- [x]  Fix Export download button focus trap.
- [x]  Fix shift tab functionality for navigation modal `skip` button clicked.

### Steps To Reproduce

First 3 items are testable from the loading package page
Other items are testable from footer panel package page
